### ISSUE TITLE
Light host didn't run.

### ISSUE THREAD

**Issue by mix0000** [rolandoislas/LightHost#16]
Using last release, and double click on app, its dont wanna open, just closing momentally, tried run as adminstrator and use compatability mode.

Pls. help

**Comment by Flogdhooft** [comment]
yo, 

I have the same problem as you on Light Host, after installing and configuring it, when I start the .exe Light Host doesn't start anymore.

I started my research and I found a youtube video that deals with the subject (between 1:24 and 2:35) but I'm not entirely satisfied : 
https://www.youtube.com/watch?v=Cx4nYuiW2yk

I also discovered that the problem was caused by the location of the VSTPlugins, if we change their location while the application is open, Light Host will no longer launch the next time we start our computer. 
So we need to remove the old options in order to reinstall Light Host properly and configure the VSTPlugins again with the new path.

Here's my solution that worked for me:

I go to C:\Users\(Username)\AppData\Roaming\Light Host to cut the options file named "Light Host.settings" and paste it temporarily on my desktop to save it.
Now I reboot my PC and restart the Light Host application, everything works as expected but we no longer have our options configured so we have to set everything up again.
So I modify the file "Light Host.settings" left on my desktop, which contains all my old options, I open it with the Notepad and I modify the paths of my VSTPlugins included in the file by the new ones. Each path needs to be updated!
Finally, I just need to move the options file from my desktop to the previously described path C:\Users\(Username)\AppData\Roaming\Light Host and the options should be applied at the next reboot.

After that, everything should be back to the way it was before, Light Host starts again and the old options have been preserved!
Hoping that was clear and that it helped you. ;)

**Comment by Flogdhooft** [comment]
> Using last release, and double click on app, its dont wanna open, just closing momentally, tried run as adminstrator and use compatability mode.
> 
> Pls. help

yo,

I have the same problem as you on Light Host, after installing and configuring it, when I start the .exe Light Host doesn't start anymore.

I started my research and I found a youtube video that deals with the subject (between 1:24 and 2:35) but I'm not entirely satisfied :
https://www.youtube.com/watch?v=Cx4nYuiW2yk

I also discovered that the problem was caused by the location of the VSTPlugins, if we change their location while the application is open, Light Host will no longer launch the next time we start our computer.
So we need to remove the old options in order to reinstall Light Host properly and configure the VSTPlugins again with the new path.

Here's my solution that worked for me:

I go to C:\Users(Username)\AppData\Roaming\Light Host to cut the options file named "Light Host.settings" and paste it temporarily on my desktop to save it.
Now I reboot my PC and restart the Light Host application, everything works as expected but we no longer have our options configured so we have to set everything up again.
So I modify the file "Light Host.settings" left on my desktop, which contains all my old options, I open it with the Notepad and I modify the paths of my VSTPlugins included in the file by the new ones. Each path needs to be updated!
Finally, I just need to move the options file from my desktop to the previously described path C:\Users(Username)\AppData\Roaming\Light Host and the options should be applied at the next reboot.

After that, everything should be back to the way it was before, Light Host starts again and the old options have been preserved!
Hoping that was clear and that it helped you. ;)
